stable/openvpn/README.md

```diff
@@ -146,3 +147,43 @@ And optionally (see openvpn.taKey setting):
  `/etc/openvpn/certs/pki/ta.key`
 
 Note: using mounted secret makes creation of new client certificates impossible inside openvpn pod, since easyrsa needs to write in certs directory, which is read-only.
+
+## Issues
+
+### 1. Routing / ip_forward
+
+Issue: https://github.com/helm/charts/issues/6398
```

Comment:
I'll test that out today.  If it works I definitely like that solution since the short-running init container has to run privileged but openvpn can then be run normally.  If that PR fixes the ip_forwarding I'll remove this bit from the readme

For replicating the issue, it definitely shows up in clusters deployed with RKE.  As I understand it it's something to do with the way that RKE spins up kubelet in a container.

Reply:
The init container works for me on an RKE deployed cluster.  Updated the readme to suggest setting that value instead of the workaround

Reply:
cool, thanks for checking! then we'd just have to make sure that PR gets merged before this one.